set.mm

```diff
@@ -147407,36 +147521,28 @@ computer programs (as last() or lastChar()), the terminology used for
     $( The value of a prefix operation for a length argument not in the range
        of the word length is the empty set.  (This is due to our definition of
        function values for out-of-domain arguments, see ~ ndmfv ).
-       (Contributed by AV, 30-Nov-2022.) $)
+       (Contributed by AV, 30-Nov-2022.)  (Proof shortened by AV,
+       3-Dec-2022.) $)
```

Comment:
I think the "shortened" tag is not needed here considering that it is the same person and three days apart. Similarly at other places, try to reduce tag inflation.

Reply:
Done